README.md

```diff
@@ -1,22 +1,97 @@
-# Q Election Votes
-
-[![Greenkeeper badge](https://badges.greenkeeper.io/nzzdev/Q-election-votes.svg)](https://greenkeeper.io/)
+# Q Election Votes [![Build Status](https://travis-ci.com/nzzdev/Q-election-votes.svg?branch=dev)](https://travis-ci.com/nzzdev/Q-election-votes) [![Greenkeeper badge](https://badges.greenkeeper.io/nzzdev/Q-election-votes.svg)](https://greenkeeper.io/)
 
 **Maintainer**: [benib](https://github.com/benib)
 
 Q election votes is one tool of the Q toolbox to display results of parliamentary elections.
+Test it in the demo: https://editor.q.tools
+
+## Table of contents
+
+- [Installation](#installation)
+- [Configuration](#configuration)
+- [Development](#development)
+- [Testing](#testing)
+- [Tool implementation details](#tool-implementation-details)
+- [Features](#features)
+- [Options](#options)
+
+## Installation
+
+```bash
+$ nvm use
+$ npm install
+$ npm run build
+```
+
+## Configuration
+
+No configuration is needed for this tool.
```

Comment:
@philipkueng @benib shall we remove this section if no configuration is needed?- Or keep it to document that there are no env vars needed?- What do you think @philipkueng

Reply:
I think it should be removed when not used.

Reply:
we decided to keep it